share/ansible_navigator/utils/image_introspect.py

```diff
@@ -185,40 +222,55 @@ def parse(command: Command):
 
 
 class AnsibleVersion(CmdParser):
-    """collect ansible version"""
+    """Define the ansible version collector."""
 
     @property
     def commands(self) -> List[Command]:
-        """generate the command"""
+        """Define the ansible command to get the version.
+
+        :returns: The defined command
+        """
         return [Command(id="ansible_version", command="ansible --version", parse=self.parse)]
 
     @staticmethod
     def parse(command: Command) -> None:
-        """parse"""
+        """Parse the output of the ansible command.
+
+        :param command: The result of running the command
+        """
         version = command.stdout.splitlines()[0].split(" ", 1)[1].strip()[1:-1]
         command.details = version
 
 
 class OsRelease(CmdParser):
-    """collect os release info"""
+    """Define the OS release information collector."""
 
     @property
     def commands(self) -> List[Command]:
-        """generate the command"""
+        """Define the command to collect os release information.
+
+        :returns: The defined command
+        """
         return [Command(id="os_release", command="cat /etc/os-release", parse=self.parse)]
 
     def parse(self, command) -> None:
-        """parse"""
+        """Parse the output of the cat command.
+
+        :param command: The result of running the command
+        """
         parsed = self.splitter(command.stdout.splitlines(), "=")
         command.details = parsed
 
 
 class PythonPackages(CmdParser):
-    """collect python packages"""
+    """Define the installed python package collector."""
```

Comment:
It's a class docstring
```suggestion
    """Installed python package collector."""
```